tools/pyvmi/module-list.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+"""module-list.py.
+
+Usage:
+  module-list.py [options] <vm_name>
+
+Options:
+  -h --help     Show this screen.
+"""
+
+import logging
+import sys
+from docopt import docopt
+from contextlib import contextmanager
+
+from libvmi import Libvmi, VMIOS, PageMode
+
+
+def init_logger():
```

Comment:
Both `init_logger` and `pause` are duplicated in 2 modules, maybe better to put them in a common module (common.py, utils.py)?